src/base/static/js/routes.js

```diff
@@ -115,6 +118,16 @@ Shareabouts.Util = Util;
       this.appView.newPlace();
     },
 
+    addInvite: function() {
+      console.log("tracking user, mixpanel:", mixpanel);
+      mixpanel.track("user invited", {
+        name: "john",
+        id: 123,
+        provider_id: "asdfqwer",
+      });
+      this.viewMap();
+    },
```

Comment:
I'm thinking we can have this route open up a modal over the map, where if the user is logged in, they would see a message like "Your invite request has been received! You will be added to the team shortly." Then they would click "ok" to close the window and go back to the map.

But if they aren't logged in, the modal would show a message like "Please log in by clicking the 'sign in' button in the upper right corner. Then click the invite link again to gain access to the team."